applications/StructuralMechanicsApplication/custom_constitutive/plasticity_isotropic_kinematic_j2.h

```diff
@@ -0,0 +1,399 @@
+// KRATOS  ___|  |                   |                   |
+//       \___ \  __|  __| |   |  __| __| |   |  __| _` | |
+//             | |   |    |   | (    |   |   | |   (   | |
+//       _____/ \__|_|   \__,_|\___|\__|\__,_|_|  \__,_|_| MECHANICS
+//
+//  License:		 BSD License
+//					 license: structural_mechanics_application/license.txt
+//
+//  Main authors:    Fernando Rastellini
+//  Collaborators:   Riccardo Rossi
```

Comment:
i did nothing here...so you don't need to add me

Reply:
you provided advise :)